UPGRADE-1.x.md

```diff
@@ -1,6 +1,10 @@
 UPGRADE 1.x
 ===========
 
+## Deprecated AbstractTypedWriterTestCase class
+
+The `Test\Writer\AbstractBlockServiceTest` class is deprecated. Use `Test\AbstractTypedWriterTestCase` instead.
```

Comment:
The first class is still the wrong class.

Should be `AbstractTypedWriterTestCase`.